src/Renderer/Camera.js

```diff
@@ -35,18 +37,16 @@ Camera.prototype.resize = function resize(width, height) {
     this.height = height;
     this.ratio = width / height;
 
-    this.camera3D.aspect = this.ratio;
-    this.camera3D.updateProjectionMatrix();
-
-    if (this.cameraHelper) {
-        var dir = new THREE.Vector3(0, 0, -1);
-        var quaternion = new THREE.Quaternion();
-        quaternion.setFromAxisAngle(new THREE.Vector3(0, 1, 0), this.HFOV / 2);
-        dir.applyQuaternion(quaternion);
-
-        this.arrowHelper.setDirection(dir);
-        this.cameraHelper.update();
+    if (this.camera3D.aspect != this.ratio) {
+        this.camera3D.aspect = this.ratio;
+        if (this.camera3D.isOrthographicCamera) {
+            const halfH = (this.camera3D.right - this.camera3D.left) * 0.5 / this.ratio;
```

Comment:
Err sorry I was wrong :)
`height` is a pixel value while `this.camera3D.right` - `this.camera3D.left` are using CRS value, ie: the geographic area visible that will be drawn on the screen.